README.md

```diff
@@ -5,143 +5,62 @@ A nextflow pipeline with a GMS touch for running the ARTIC network's fieldbioinf
 ------------
 #### Major changes
 
-* The illumina and nanopore tracks automatically run pangolin
+* The illumina and nanopore tracks automatically run pangolin and nextclade.
+* Generates report for base changes.
 
-##### gms-artic in ngp-gms
+PLEASE FIND THE DETAILS HERE: [Wiki page](https://github.com/JD2112/gms-artic/wiki/User-information)
+###### 1. gms-artic in ngp-gms
 
-###### 1. login to the system
-ssh xxxxxx@xx.x.gu.se \
-password \
-$
-
-==NOTE: sge process is not supported on the /gms-storage-aws/, so need to run the pipeline locally from /home==
---------------
-#### # Things to consider before running the pipeline
-1. Update the Singularity container once in a week
-2. Pangolin version is updating almost every week/twice per week.
-3. For different nanopore pipelines, need to use different primers;
-primer-schemes are designed as per Nanopore protocol, need to choose the right one, before executing the run, 
-```--scheme-directory primer_schemes/[artic/midnight/eden]/nCoV-2019/[Version number]/```
-4. Separate Pangolin Singularity container required for the Nanopore analysis. Need to update the container regularly.
-```docker pull staphb/pangolin:latest
-https://hub.docker.com/r/staphb/pangolin/tags?page=1&ordering=last_updated
+##### Running autoloader to check new incoming data and metadata file
+```
+./runner.sh
 ```
-5. Available Containers
-    A. Cloud - 
-    i. Singularity - Illumina, Nanopore, Pangolin
-    ```
-    singularity pull library://jd2112/gms-artic-illumina/gms-artic-illumina.sif:latest
-    singularity pull library://jd2112/gms-artic-illumina/gms-artic-nanopore.sif:latest
-    singularity pull library://jd2112/gms-artic-illumina/pangolin:latest
-    ```
-    ii. Docker - Illumina, Nanopore, Pangolin
-    ```
-    docker pull jd21/gms-artic_illumina:latest
-    docker pull jd21/gms-artic_nanopore:latest
-    docker pull jd21/pangolin:latest
-    ```
-    B. Local
-    iii.Singularity 
-    iv. Conda
```

Comment:
Some of details of this section. E.g. logging in and updating images seems to be completely deleted now. Whilst not necessary in the README. Consider porting as much of the details as possible into a side document

Reply:
I can put them to the wiki page.